README.md

```diff
@@ -17,49 +17,49 @@ Dive into the world of Ruby-based structured extraction, by OpenAI's function ca
 
     - [Python](https://www.github.com/jxnl/instructor)
     - [TS/JS](https://github.com/instructor-ai/instructor-js/)
+    - [Ruby](https://github.com/instructor-ai/instructor-rb)
     - [Elixir](https://github.com/thmsmlr/instructor_ex/)
 
     If you want to port Instructor to another language, please reach out to us on [Twitter](https://twitter.com/jxnlco) we'd love to help you get started!
 
 ## Usage
 
-```rb
-# Todo, change to ruby
-import Instructor from "@instructor-ai/instructor";
-import OpenAI from "openai"
-import { z } from "zod"
-
-const UserSchema = z.object({
-  age: z.number(),
-  name: z.string()
-})
-
-type User = z.infer<typeof UserSchema>
-
-const oai = new OpenAI({
-  apiKey: process.env.OPENAI_API_KEY ?? undefined,
-  organization: process.env.OPENAI_ORG_ID ?? undefined
-})
-
-const client = Instructor({
-  client: oai,
-  mode: "FUNCTIONS" # or "TOOLS" or "MD_JSON" or "JSON"
-})
-
-const user = await client.chat.completions.create({
-  messages: [{ role: "user", content: "Jason Liu is 30 years old" }],
-  model: "gpt-3.5-turbo",
-  response_model: { schema: UserSchema }
-})
-
-console.log(user)
-// { age: 30, name: "Jason Liu" }
+export your OpenAI API key:
+
+```bash
+export OPENAI_API_KEY=sk-...
+```
+
+Then use Instructor to extract structured data from text in Ruby:
+
+```ruby
+require 'instructor'
+
+class UserDetail < Instructor::Model
+  params do
+    required(:name).filled(:string)
+    required(:age).filled(:integer)
+  end
+end
+
+client = Instructor::OpenAI::Client.new
+
+user = client.chat(
+  parameters: {
+    model: 'gpt-3.5-turbo',
+    messages: [{ role: 'user', content: 'Extract Jason is 25 years old' }]
+  },
+  response_model: UserDetail
+)
+
+puts(user.inspect)
+#=> #<Dry::Validation::Result{:name=>"Jason", :age=>25} errors={}>
```

Comment:
Note the response object is a `Dry::Validate::Result`. We'd probably want an instance of the UserDetail class instead.